sqlglot/dialects/bigquery.py

```diff
@@ -162,6 +162,11 @@ def _build_timestamp(args: t.List) -> exp.Timestamp:
     return timestamp
 
 
+def _build_parse_datetime(args: t.List) -> exp.ParseDatetime:
+    this = build_formatted_time(exp.ParseDatetime, "bigquery")([seq_get(args, 1), seq_get(args, 0)])
```

Comment:
Nit: can't we inline this: `build_formatted_time(exp.ParseDatetime, "bigquery")` in L640?